app/services/find/course_search_service.rb

```diff
@@ -0,0 +1,256 @@
+module Find
```

Comment:
I don't think we need to add **_another_** course search service just for Find. Currently, we have the one https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/services/course_search_service.rb which powers the current Find search and the public api search. 

https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/controllers/api/v3/courses_controller.rb#L9 this just delegates to the one above.